UvaSoftware.Scanii/UvaSoftware.Scanii.csproj

```diff
@@ -17,10 +17,8 @@
     <IsPackable>true</IsPackable>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.6.2"/>
```

Comment:
any idea why some of these were removed?

Reply:
They were removed because they weren't actually being used by the project and where ultimately causing issues once I switched it over to target netstandard2.0 (e.g. NUnit3TestAdapter has a dependency on dotnet core)